starter-code/scripts/controllers/articleController.js

```diff
@@ -23,6 +24,7 @@
   };
 
   // COMMENT: What does this method do?  What is it's execution path?
+  // First, this method matches what is selected in the author filter dropdown to all articles by author. Then it uses the articleController path to find the articles and render them to the page.
```

Comment:
Correct, but here's a more detailed explanation: this method calls Article.findWhere, which queries the database for articles with an author matching the ctx.params.authorName property (passed via the route being hit in articleView.handleFilters). It passes in authorData, defined above, as a callback to Article.findWhere, so after the query, an articles property is attached to our context object containing all of the articles that match our query. Then, the next function in the middleware chain is called (articleController.index).